Fruitties/iosApp/iosApp/CartView.swift

```diff
@@ -42,22 +42,22 @@ struct CartView : View {
                 }.padding()
             }
             if (expanded) {
-                CartDetailsView(dataRepository: dataRepository)
+                CartDetailsView(mainViewModel: mainViewModel)
             }
         }
     }
 }
 
 struct CartDetailsView: View {
-    let dataRepository: DataRepository
+    let mainViewModel: MainViewModel
     @State
-    private var details: CartDetails = CartDetails(items: [])
+    private var cartUiState: CartUiState = CartUiState(cartDetails: [])
```

Comment:
Thanks! I'll update to match the style.

Reply:
I made some changes to remove the UIModel. This means I'm accessing the KMP ViewModel directly with variables named in Kotlin. I now have a mix of capitalization: KMP object properties use names like cartUiState and SwiftUI State properties use names like cartUIState. It probably looks a little weird for everyone, but updating this would require updating Android code (to be considered separately).